app/search_builders/curation_concerns/work_search_builder.rb

```diff
@@ -1,5 +1,9 @@
 module CurationConcerns
   class WorkSearchBuilder < ::SearchBuilder
     include CurationConcerns::SingleResult
+    include CurationConcerns::FilterSuppressed
+
+    # work show page should not filter out suppressed works set by the base search builder used for catalog searches
+    self.default_processor_chain -= [:only_active_works]
```

Comment:
I added a comment describing the need... the base search builder filters out suppressed works from the SOLR query. So when you try to go to a Work show page, you have to eliminate the filter.